cypress/integration/pages/home_page_spec.ts

```diff
@@ -0,0 +1,28 @@
+/// <reference types="cypress" />
+export {};
+
+describe('Test Home Page', () => {
+  it('loads successfully', () => {
+    cy.visit('/');
+  });
+  it('renders the hero title', () => {
+    cy.contains('Find, Share and Publish Quality Data with Datahub');
+  });
+  it('displays a search form', () => {
+    cy.get('form').type('dataset').contains('Search').click();
+  });
+  it('submits the search form', () => {
+    cy.get('form').find('[type="text"]').type('my-dataset');
+    cy.get('form').submit();
+  });
+  it('shows the recent datasets', () => {
+    cy.contains('Recent Datasets');
+  });
+  it('returns the expected number of datasets', () => {
+    cy.get('.recent')
+      .find('div')
+      .should(($div) => {
+        expect($div).to.have.length(3);
```

Comment:
Against which DMS backend are we testing? If against mocks then I'd expect length of 2.

Reply:
Testing against CKAN API and not the mocks.

Reply:
@lauragift21 ok, which CKAN API? Where can we set it up?